config/kubernetes/org.yaml

```diff
@@ -1281,6 +1281,20 @@ teams:
     - lavalamp
     - sttts
     privacy: closed
+  cloud-provider-aws-admins:
```

Comment:
Can this be moved to https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-cloud-provider/teams.yaml? That way you can approve PRs touching this team directly :)

Reply:
+1

Reply:
Yes, good call let me update

Reply:
updated